test/resolve-config.test.js

```diff
@@ -8,16 +8,21 @@ test('Returns user config', t => {
   const gitlabApiPathPrefix = '/api/prefix';
   const assets = ['file.js'];
   const postComments = true;
+  const labels = false;
 
   t.deepEqual(
-    resolveConfig({gitlabUrl, gitlabApiPathPrefix, assets, postComments}, {env: {GITLAB_TOKEN: gitlabToken}}),
+    resolveConfig({gitlabUrl, gitlabApiPathPrefix, assets, postComments, labels}, {env: {GITLAB_TOKEN: gitlabToken}}),
     {
       gitlabToken,
       gitlabUrl,
       gitlabApiUrl: urlJoin(gitlabUrl, gitlabApiPathPrefix),
       assets,
       milestones: undefined,
       successComment: undefined,
+      failTitle: 'The automated release is failing 🚨',
+      failComment: undefined,
+      labels: false,
+      assignee: undefined,
     }
```

Comment:
Maybe we should define an object with the default values and reuse that across all test cases to reduce duplication?

Something like this:

```js
const defaultOptions = {
      // ...
      successComment: undefined,
      failTitle: 'The automated release is failing 🚨',
      assignee: undefined,
      // ...
}

// ...

t.deepEqual(
    resolveConfig({gitlabUrl, gitlabApiPathPrefix, assets, postComments, labels}, {env: {GITLAB_TOKEN: gitlabToken}}),
    {
      ... defaultOptions,
      gitlabApiUrl: urlJoin(gitlabUrl, gitlabApiPathPrefix),
    }
```